lib/panoramix/query.ex

```diff
@@ -453,11 +453,15 @@ defmodule Panoramix.Query do
   end
   defp dimension_or_extraction_fn({:lookup, _, args}) do
     case args do
-      [from_dimension, lookup_name] ->
+      [from_dimension, lookup_name | maybe_opts] ->
+        opts = case maybe_opts do
+                 [] -> []
+                 [opts] -> opts
```

Comment:
Would that work if `maybe_opts` contains more than 1 option in a list?

Reply:
Yes, I added a test that exercises that.